src/Core/src/Handlers/WebView/WebViewHandler.iOS.cs

```diff
@@ -511,7 +511,7 @@ bool LoadFile(string url)
 			}
 			catch (Exception)
 			{
-				Console.WriteLine($"Could not load {url} as local file");
+				MauiContext?.CreateLogger<WebViewHandler>()?.LogWarning($"Could not load {url} as local file");
```

Comment:
Instead of formatting the string in-line, you can use the logging system's formatter:

```suggestion
				MauiContext?.CreateLogger<WebViewHandler>()?.LogWarning("Could not load {url} as local file", url);
```

This enables structured loggers to log the URL as a data item instead of as just a plain string mixed into a message.